src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php

```diff
@@ -124,3 +197,126 @@ protected function setAttributeValue($object, $attribute, $value, $format = null
         $object->$attribute = $value;
     }
 }
+
+class DummyCollection
+{
+    /** @var DummyChild[] */
+    public $children;
+}
+
+class DummyChild
+{
+    public $bar;
+}
+
+class SerializerCollectionDummy implements \Symfony\Component\Serializer\SerializerInterface, \Symfony\Component\Serializer\Normalizer\DenormalizerInterface
+{
+    /** @var \Symfony\Component\Serializer\Normalizer\DenormalizerInterface */
```

Comment:
please review 24b6848ea7 and report any mistake

Reply:
Looks good to me